chapters/operatorsandexpressions.tex

```diff
@@ -1342,20 +1380,24 @@ \subsection{Discrete-Time Expressions}\doublelabel{discrete-time-expressions}
 For an equation \lstinline!expr1 = expr2! where neither expression is of base type
 \lstinline!Real!, both expressions must be discrete-time expressions. For record
 equations the equation is split into basic types before applying this
-test. {[}\emph{This restriction guarantees that the} \lstinline!noEvent()!
-\emph{operator cannot be applied to} \lstinline!Boolean!\emph{,} \lstinline!Integer!\emph{,}
-\lstinline!String!\emph{, or} \lstinline!enumeration! \emph{equations outside of a} \lstinline!when!\emph{-clause, because
-then one of the two expressions is not discrete-time}{]}
+test.
+
+\begin{nonnormative}
+This restriction guarantees that the \lstinline!noEvent()! operator cannot be applied to \lstinline!Boolean!, \lstinline!Integer!, \lstinline!String!, or \lstinline!enumeration!
+equations outside of a \lstinline!when!-clause, because then one of the two expressions is not discrete-time.
+\end{nonnormative}
 
 Inside an if-expression, \lstinline!if!-clause, \lstinline!while!-statement or \lstinline!for!-clause, that
 is controlled by a non-discrete-time (that is continuous-time, but not
 discrete-time) switching expression and not in the body of a
 \lstinline!when!-clause, it is not legal to have assignments to discrete variables,
 equations between discrete-time expressions, or real elementary
-relations/functions that should generate events. {[}\emph{This
-restriction is necessary in order to guarantee that there all equations
-for discrete variable are discrete-time expressions, and to ensure that
-crossing functions do not become active between events.}{]}\\
+relations/functions that should generate events.
+
+\begin{nonnormative}
+The restriction above is necessary in order to guarantee that there all equations for discrete variable are discrete-time expressions, and to ensure that crossing
```

Comment:
Just noticed that it was wrong before as well. "to guarantee that there all equations"->to guarantee that all equations"

Reply:
Fixed.